style.css

```diff
@@ -254,3 +256,172 @@ textarea:hover {
 input:hover {
   border: 1px solid #253858;
 }
+
+@media screen and (min-width: 768px) {
+  .main {
+    background: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-color: #fff;
+    border-bottom-left-radius: 100px;
+  }
```

Comment:
- [ ] After making changes please 🙏🏽 ensure that your design matches the Figma design, please correct the margins, padding, and position of the elements in the header section.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/236583504-9ee94cda-f8c7-4fed-8d97-8771f411943f.png) | ![image](https://user-images.githubusercontent.com/58642949/236583526-61680819-d1ac-40ba-9dd5-630a24606bf5.png) |